apps/flux-system/base/image-automation-components.yaml

```diff
@@ -1966,17 +1978,150 @@ metadata:
   labels:
     app.kubernetes.io/instance: flux-system
     app.kubernetes.io/part-of: flux
-    app.kubernetes.io/version: v0.18.3
+    app.kubernetes.io/version: v0.25.0
   name: image-reflector-controller
   namespace: flux-system
 ---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
this needs to be manually removed see https://github.com/hmcts/cnp-flux-config#upgrading-flux-v2